index.qmd

```diff
@@ -0,0 +1,200 @@
+---
+pagetitle: "Philly Social Justice Hackathon"
+page-layout: custom
+section-divs: false
+toc: false
+anchor-sections: true # not working?
+aliases:
+  - index.html
+execute: 
+  echo: false
+  output: asis
+  warning: false
+---
+
+
+```{r workspace}
+#| include: false
+library(htmltools)
+library(shiny) # markdown()
+library(glue)
+
+general_card_div <- function(..., lg = 3, md = 2, sm = 2, xs = 1) {
+  class_info <- 
+    paste(
+      "card",       # card class
+      # about card
+      "rounded-3",  # border-radius = 0.3rem
+      "m-2",        # margin all sides: 0.5rem
+      # about grid placement
+      glue("g-col-lg-{12/lg}"), # when >=1200px use {lg} cols per card (ex. 3 across)
+      glue("g-col-md-{12/md}"), # when >= 992px use {md} cols per card (ex. 2 across)
+      glue("g-col-sm-{12/sm}"), # when >= 768px use {sm} cols per card (ex. 2 across)
+      glue("g-col-{12/xs}")     # when  < 576px use {xs} cols per card (ex. 1 across)
+    )
+  
+  div(
+    class = class_info,
+    style = "background: #8f4b99;",
+    ...
+  )
+}
+```
+
+<!-- HERO BANNER -->
+
+::: {.content-block}
+```{r}
+a(
+  class="preview-link",
+  href="https://www.eventbrite.com/e/860575954847",
+  target="_blank",
+  p(
+    class="hero-link content-block",
+    HTML("Pre-Hackathon Events: Join us for Open Source Tech in Legal Aid this Thurs, March 21, 2024, at the Drexel Kline School of Law! Learn about challenge projects and data to get ready for the #SJHPHL Hackathon in April")
+  )
+)
+```
+:::
+
+::: {.hero-banner .text-center}
+```{r hero-banner}
+h3(
+    class="text-left",
+    HTML("Philly's 2nd")
+)
+h1(
+  class = "text-left hero-title",
+  HTML("Social Justice Hackathon")
+)
+h3(
+    class="text-left",
+    HTML("April 5th-7th @ Drexel Kline School of Law")
+)
+```
+:::
+
+<!-- /HERO BANNER -->
+
+:::::: {.content-block}
+:::{.hero-text .content-block}
+  The Philadelphia Social Justice Hackathon will be held at the Drexel Law School from April 5-7, 2024, to bring students, programmers, and other “civic-minded hackers” together – to work in collaboration with justice advocates, public servants, community legal providers, and members of the community.
```

Comment:
From [this slack thread](https://codeforphilly.slack.com/archives/C06DXBH1HE2/p1711299903315159?thread_ts=1711248924.227479&cid=C06DXBH1HE2)

```suggestion
Students, programmers, “civic-minded hackers”, faculty, justice advocates, public servants, community legal providers, writers, story-tellers, and members of the Philly community will all come together to design creative solutions to respond to specific justice needs of residents and neighborhoods most impacted by social, economic, and environmental injustices in Philadelphia.
```